paper-radio-button.html

```diff
@@ -153,9 +152,6 @@
       }
 
       /* disabled state */
-      :host([disabled]) {
-        pointer-events: none;
-      }
```

Comment:
[IronControlState already handles this.](https://github.com/PolymerElements/iron-behaviors/blob/9ffe111/iron-control-state.html#L87)